src/esp/sim/Simulator.cpp

```diff
@@ -573,10 +573,9 @@ void Simulator::reconfigureReplayManager() {
   resourceManager_->setRecorder(gfxReplayMgr_->getRecorder());
 
   // provide Player callback to replay manager
-  gfxReplayMgr_->setPlayerCallback([this](auto&& PH1, auto&& PH2) {
-    loadAndCreateRenderAssetInstance(std::forward<decltype(PH1)>(PH1),
-                                     std::forward<decltype(PH2)>(PH2));
-  });
+  gfxReplayMgr_->setPlayerCallback(
+      std::bind(&esp::sim::Simulator::loadAndCreateRenderAssetInstance, this,
```

Comment:
why was this change needed? Isn't std::bind less performant here?

Reply:
This was giving me an error for some reason. I got std::bind from the current master branch of habitat.